components/accept-financial-contributions/ContributionCategoryPicker.js

```diff
@@ -153,25 +139,15 @@ class ContributionCategoryPicker extends React.Component {
                 width={[null, 280, 312]}
               >
                 <Flex flexDirection="column" justifyContent="center" alignItems="center">
-                  <Image src={skateboardingIllustration} alt={intl.formatMessage(this.messages.host)} />
+                  <Image src={acceptHostIllustration} alt={intl.formatMessage(this.messages.host)} />
                   <Link
                     route="accept-financial-contributions"
                     params={{
                       slug: router.query.slug,
                       path: 'host',
                     }}
                   >
-                    <StyledButton
-                      fontSize="13px"
-                      buttonStyle="dark"
-                      minHeight="36px"
-                      mt={[2, 3]}
-                      mb={3}
-                      px={3}
-                      onClick={() => {
-                        this.props.onChange('path', 'host');
-                      }}
-                    >
+                    <StyledButton fontSize="13px" buttonStyle="dark" minHeight="36px" mt={[2, 3]} mb={3} px={3}>
```

Comment:
No `onClick`?

Reply:
It's wrapped in a Link which pushes the route, and we get the route params from the query so we don't need to set the state when we click.

Reply:
I missed that, thanks!